conf.d/riak.yaml.example

```diff
@@ -2,6 +2,8 @@ init_config:
 
 instances:
   - url: http://127.0.0.1:8098/stats
+    # cacert: /path/to/ca.pem
+    # disable_cert_verify: true
```

Comment:
Since the default value in the code is `False`, could you use `false` in the configuration example ?